src/SQLChecks/Tests/SQLInstance.tests.ps1

```diff
@@ -55,4 +55,46 @@ Describe "Sysadmins" -Tag Sysadmins {
     It "Correct list of sysadmins on $serverInstance" {
         (Test-Sysadmins $Config).Count | Should -Be 0
     }
-}
\ No newline at end of file
+}
+
+Describe "Tempdb Configuration" -Tag TempdbConfiguration {
+	
+    $NumberOfFiles = $config.TempDBConfiguration.NumberOfFiles
+    $TotalSizeMB = $config.TempDBConfiguration.TotalSizeMB
+	
+    It "Correct number of TempDB files on $serverInstance" {
+        (Get-TempDBConfiguration -Config $Config).NumberOfFiles | Should Be $NumberOfFiles
+    }
+
+    It "Correct TempDB size on $serverInstance" {
+        (Get-TempDBConfiguration -Config $Config).TotalSizeMB | Should Be $TotalSizeMB
+    }
+
+}
+
+Describe "Resource Governor" -Tag ResourceGovernor {
```

Comment:
What else could be missing so that the pools/groups are configured correctly but RG doesn't work?  Maybe for future checks, I *think* you'd need to check:

- Is RG enabled on the server?
- Is the classifier correct?

Reply:
This info lives in sys.resource_governor_configuration - can add these to the config and update documentation as well